src/middlewared/middlewared/restful.py

```diff
@@ -21,26 +21,35 @@ async def authenticate(middleware, req):
     if auth is None:
         raise web.HTTPUnauthorized()
 
+    username = password = None
+    if auth.startswith('Authorization: Basic '):
```

Comment:
Hmm, we are already getting `Authorization` header, not sure why we would check for another Authorization within that header